src/components/modal/Modal.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  ModalTitleWrapper,
+  ModalTitie,
+  ModalSideHeader,
+} from '../../styles/styledComponents/folderStyled';
+import { modalSubTitle } from '../../assets/utils/modalSubTitle';
+import { DeleteFolderModal } from './DeleteFolderModal';
+import { DeleteLinkModal } from './DeleteLinkModal';
+import { SharingModal } from './SharingModal';
+import { AddModal } from './AddModal';
+import { AddFolderModal } from './AddFolderModal';
+import { EditFolderModal } from './EditFolderModal';
+const MODALNAME = {
```

Comment:
대문자 스네이크 케이스를 쓰는것도 추천드려요!
```suggestion
const MODAL_NAME = {
```